library/src/main/java/com/getstream/sdk/chat/utils/frescoimageviewer/MultiTouchViewPager.java

```diff
@@ -7,7 +7,7 @@
 import androidx.annotation.NonNull;
 import androidx.viewpager.widget.ViewPager;
 
-class MultiTouchViewPager extends ViewPager {
+public class MultiTouchViewPager extends ViewPager {
```

Comment:
This was needed for ViewBinding to be able to generate bindings that contain this class.